### PR TITLE
[NFC][CursorInfo] Temporarily disable some doc comment check lines

### PR DESCRIPTION
**Pull request by zixu-w**
Pull request apple/llvm-project#4442 brings in a change to
`RawComment::getFormattedText` that removes spurious new lines
and whitespaces at the end of block comments. It breaks the
`cursor_symbol_graph_objc` test which is assuming the old behavior.
Temporarily disable the relevant check lines in the test to merge the
llvm change, and then fix the test properly and switch to the new
`getFormattedLines` in SymbolGraphGen.

### CODE REVIEW

**Comment by bnbarham** [comment]
There's quite a bit of cherry-picking that will be required at the moment :(
swift `main` corresponds to `stable/20211026`
swift `release/5.7` corresponds to `apple/swift/5.7`
and I'm currently working on the rebranch where `main` will eventually switch to `stable/20220421` (currently `rebranch` corresponds to `stable/20220421`)

TLDR: You'll want this change in `main` and `release/5.7` + the LLVM change in `stable/20211026` and `apple/swift/5.7`. `main` is automerged to `rebranch` so we'll also need the LLVM change in `stable/20220421`, but I can handle that if needed.

**Review by bnbarham — APPROVED** [review]

**Comment by zixu-w** [comment]
@swift-ci Please smoke test

**Comment by zixu-w** [comment]
Linux test failed with git clone errors... Trying again
@swift-ci Please smoke test Linux platform

**Review by daniel-grumberg — APPROVED** [review]
